boards/arm/nucleo_h743zi/nucleo_h743zi_defconfig

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Any reason the ARM_MPU is not defined in the default configuration?

Reply:
@ioannisg I think we still miss a statement if MPU should be part of default board configuration. I'm open to do it, but then it should be documented (here for instance: https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#general-recommendations)

Reply:
Good point. Some other ARM boards have this option enabled, while some others do not. I based my initial config on `stm32h747i_disco`, which does not have `CONFING_ARM_MPU=y`.

Reply:
I'd like to make it default actually, so I prefer that you add it.
That is a CM7 - should be on high-end cortex-m, thus, OK to run by default with advanced ARM features.
but I agree it is not yet default so leave it without MPU if you insist.
Note, that enabling ARM MPU you enable building/running for a wider range of samples and tests that require user mode, stack protection, etc.

Reply:
@ioannisg @erwango what do you guys prefer then?

Reply:
@gmarull Ok to enable

Reply:
Ok, added the option (and tested that examples work fine)

Reply:
thanks!